Improve 'no-nested-template-literals': do not report on multiline literal (#326)
SonarSource/eslint-plugin-sonarjs: docs/rules/no-nested-template-literals.md

let apples = color ? `${count} ${color}` : count;
let message = `I have ${apples} apples`;
```

## Exceptions

> The case when both the start and end of nesting template is on different line from nested literal, is ignored.

Reviewer: ```suggestion
Nested templates are ignored when the nesting one is on a different line from the nested one.
```
